src/api.ts

```diff
@@ -21,4 +21,4 @@ function get<T>(url: string): Promise<T> {
 
 export const getTodos = () => get<Todo[]>('/todos');
 
-export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+export const getUser = (userId: number | null) => get<User>(`/users/${userId}`);
```

Comment:
The 'getUser' function may not behave as expected when 'userId' is null. The resulting URL will become '/users/null' which likely does not match any route on the server. Consider adding a guard clause to handle the case when 'userId' is null.